spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe CustomersController, type: :feature do
+    it 'displays the customer name and a list of their items' do
+        kings = Supermarket.create(name: 'Kings', location: '123 main st')
+        customer1 = Customer.create(name: 'Bob')
+        bread = Item.create(name: 'Bread', price: 3, supermarket: kings)
+        eggs = Item.create(name: 'Eggs', price: 2, supermarket: kings)
+        CustomerItem.create(customer: customer1, item: bread)
+        CustomerItem.create(customer: customer1, item: eggs)
+
+        visit "/customers/#{customer1.id}"
+
+        expect(page).to have_content("Bob")
+        expect(page).to have_content("Bread")
+        expect(page).to have_content("Eggs")
+        expect(page).to have_content(3)
```

Comment:
We'd love to see all these assertions in within blocks. Just asserting that a `2` is not very meaningful unless we know it's in the section for Eggs. We can also get false positives on these tests is different items have the same price, or if there is a section on the page with other numbers displayed. Alternatively, if you set a text label up in your view, you can also anchor these assertions with their label like so: `expect(page).to have_content("Price: 2")`